sdk/Makefile

```
@@ -49,27 +35,39 @@ $(HIBERNATE_FOLDER): $(CLIENT_FOLDER)
 $(APP_FOLDER): $(HIBERNATE_FOLDER)
 	cd $(APP_FOLDER) && mvn clean install -DskipTests && cd -
 
-.PHONY: all
-all: $(APP_FOLDER)
+.PHONY: run-vault
+run-vault:
+	docker rm -f pvault-dev
+	docker run --init \
+		 --name pvault-dev \
+		 --add-host='host.docker.internal:host-gateway' \
+		 -p 8123:8123 \
+		 -e PVAULT_SERVICE_LICENSE=${PVAULT_SERVICE_LICENSE} \
+		 -e PVAULT_SENTRY_ENABLE=true \
+		 -e PVAULT_LOG_CUSTOMER_IDENTIFIER="piiano" \
+		 -e PVAULT_LOG_CUSTOMER_ENV="VAULT-JAVA" \
+		 -d \
+		 piiano/pvault-dev:$(VAULT_VERSION)
+	sleep 5
 
 .PHONY: test-openapi
-test-openapi: run-vault $(CLIENT_FOLDER)
-	cd $(GEN_FOLDER) && mvn -X test && cd -
+test-openapi: generate-openapi
+	cd $(GEN_FOLDER) && mvn test && cd -
```

I think it's cleaner to pass `-f` flag to tell maven to run in another folder instead of running `cd` before and after.
```suggestion
	mvn -f $(GEN_FOLDER) test
```
This change is relevant for all the other commands below too.